### PR TITLE
Refactor Alphabet to fail if alphabet file doesn't exist

### PR DESCRIPTION
**Pull request by dabinat**
Opening this as a new PR because I accidentally screwed up #2193.

### CODE REVIEW

File: native_client/ctcdecode/scorer.cpp [hunk 1]
```diff
@@ -61,8 +61,11 @@ Scorer::Scorer(double alpha,
                const std::string& lm_path,
                const std::string& trie_path,
                const std::string& alphabet_config_path)
-  : Scorer(alpha, beta, lm_path, trie_path, Alphabet(alphabet_config_path.c_str()))
 {
+    Alphabet alphabet;
+    alphabet.loadConfigFile(alphabet_config_path.c_str());
```
Comment (reuben) [inline native_client/ctcdecode/scorer.cpp]: Hm. This is masking the failure when called from `DS_EnableDecoderWithLM`, which used to catch it with a try-catch block. The best thing to do would be to refactor `Scorer` as well to move fallible initialization to a method, but I'd also be happy with just adding a throw here if `loadConfigFile` fails.

**Comment by reuben** [comment]
Oh, and please revert to 2 spaces for indentation, it's what we use in all (most?) of our C++ code.

**Comment by dabinat** [comment]
I was going to do some more work on this but it looks like #2303 makes significant changes to this part of the code, so I guess I’ll wait for that to land first and then reimplement. #2303 looks like it will make it much simpler to achieve anyway.

**Comment by lissyx** [comment]
> #2303 looks like it will make it much simpler to achieve anyway.

@reuben said on IRC that #2303 might be more tricky than expected, do you still want to hold your changes ?

**Comment by reuben** [comment]
I decided to just tackle this in #2303 since it was so close to what I was already doing anyway.

**Pull request closed without merging**